pkg/doc/did/doc.go

```diff
@@ -790,6 +795,16 @@ func (r *didKeyResolver) Resolve(id string) (*verifier.PublicKey, error) {
 	return nil, ErrKeyNotFound
 }
 
+// MapKeyTypeToKMS maps key type of DID doc to the KMS one.
+func MapKeyTypeToKMS(t string) (kms.KeyType, error) {
```

Comment:
This function doesn't belong in the did package.

Reply:
Why? The key types are defined at https://www.w3.org/TR/did-core/#public-keys spec, and we need to map them to our internal key types.